From ba6da2eef807feadfef37da08c9812a577c815ef Mon Sep 17 00:00:00 2001
From: Manaf Meethalavalappu Pallikunhi <manafm@codeaurora.org>
Date: Fri, 25 Jan 2019 22:51:16 +0530
Subject: [PATCH] thermal-engine: Add bandwidth level request APIs to thermal
 client header

Add new bandwidth perf level request APIs to thermal client library
header so that client can inlude this header library and use these
new APIs.

Change-Id: I05a0c1f04f5d5c84a27c619a6b78c92a02aef5e8
---
 thermal_client.h | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/thermal_client.h b/thermal_client.h
index b41f139..37e8731 100644
--- a/thermal_client.h
+++ b/thermal_client.h
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, The Linux Foundation. All rights reserved.
+ * Copyright (c) 2016, 2019, The Linux Foundation. All rights reserved.
  *
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted provided that the following conditions are
@@ -87,6 +87,10 @@ int thermal_client_register_callback(char *client_name, int (*callback)(int , vo
 int thermal_client_request(char *client_name, int req_data);
 void thermal_client_unregister_callback(int client_cb_handle);
 
+/* APIs for bandwidth clients to send/clear bandwidth perf levels to thermal-engine */
+int thermal_bandwidth_client_request(char *client_name, int req_data);
+void thermal_bandwidth_client_cancel_request(char *client_name);
+
 #ifdef __cplusplus
 }
 #endif
-- 
2.17.1

